SUMMARY.md

```diff
@@ -193,7 +193,7 @@
   - [Finance](libs/clients/finance/README.md)
   - [Firearm License](libs/clients/firearm-license/README.md)
   - [Clients Fishing License](libs/clients/fishing-license/README.md)
-  - [Health Insurance](libs/clients/health-insurance-v2/README.md)
+    - [Health Insurance](libs/clients/icelandic-health-insurance/health-insurance/README.md)
```

Comment:
![Screenshot 2023-05-23 at 11 00 13](https://github.com/island-is/island.is/assets/24840451/52182f29-96f0-4dd8-bcae-abd387abc1ef)

👁️

Reply:
fixed